ci-operator/step-registry/openshift-extended/guest-cluster/OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
```

Comment:
@kuiwang02 Do you have any suggestion where we could put this kinds of steps in QE. I think this step is only usable/available for QE. So if there is any available location/dir for us, it would be great.

cc @jianlinliu for some qe's steps to do some env preparation like kubeconfig changed to guest here, what's your suggestion where to put them ? Thanks!

Reply:
> I think this step is only usable/available for QE

From the steps' command shell script, I did not see which is only usable/available for QE. If other users want to test guest cluster would have different process? Note, prow ci itself is for test purpose.

The `guess-cluster` is only in hypershift install, right? If right, as my suggestion, it should be placed under `step-registry/hypershift/` as usual.

Reply:
@jianlinliu how about this ci-operator/step-registry/cucushift/installer/rehearse/hypershift ?

Reply:
I do not think that is a good place, that folder is for saving workflow and provision/deprovision chains.

Reply:
@jianlinliu  this step is only usable for qe e2e openshift-test-private.  No one else would need it.

Reply:
We consider to put it into Dev's dir too previously. but it is only used in QE's side test and only for openshift-test-private repo. So it is wired to put it in to DEV's side for they would not use it later.
It's just our pain point that no place is allowed for us to put this kinds of steps. But it is necessary for QE's e2e test.

Reply:
I have few knowledge about hypershift cluster, but if users want to consume the guest cluster (whatever openshift-test-private or other testing), have to get kubeconfig file, right? Just like what osd cluster did, it also need to place [kubeconfig](https://github.com/openshift/release/blob/master/ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh#L174-L176) in `SHARE_DIR` so that e2e testing can move forward. Personally I do not think it is weird to place the step in `hypershift` folder. If other users do not need the new steps, he/she just do not reference the steps in their workflows, this is a common rule in prow ci.

Reply:
Yes, agreed and thanks for your comments. Try to submit this step into dev's side.